Resources/templates/responsive/workshop/index.php

```diff
@@ -9,6 +9,8 @@
     'tw_image' => $meta_img
     ]);
 
+$footer_sponsors=$this->workshop->getSponsors("footer");
+
```

Comment:
I forgot to ask if we could move this to the WorkshopController.php and just pass it to the view. After that we can publish it directly!

Everything works as expected :)

Reply:
done! I was thinking the same